src/coreclr/inc/corinfo.h

```diff
@@ -2635,7 +2635,7 @@ class ICorStaticInfo
                 unsigned int           *cILOffsets,         // [OUT] size of pILOffsets
                 uint32_t              **pILOffsets,         // [OUT] IL offsets of interest
                                                             //       jit MUST free with freeArray!
-                ICorDebugInfo::BoundaryTypes *implictBoundaries // [OUT] tell jit, all boundries of this type
```

Comment:
The same typos are duplicate in number of other places in the repo. Could you please find&replace all of them?

Reply:
@jkotas done! there were quite a few, some look like auto-generated code, but I just changed every occurrence I could find